websites/T/Tidal/presence.ts

```diff
@@ -1,97 +1,188 @@
 const presence = new Presence({
-    clientId: "707985888814039040"
-  }),
-  strings = presence.getStrings({
-    play: "presence.playback.playing",
-    pause: "presence.playback.paused"
-  });
-
-function getAuthorString(): string {
-  const authors = document.querySelectorAll(
-    "#footerPlayer > div.leftColumn--yVbBY > div.dragItem--3i9Xz > div.css-124sz32.mediaInformation--1QcQT > div.mediaArtists--2pRii > a"
-  ) as NodeListOf<HTMLAnchorElement>;
-  let authorsArray: Array<HTMLAnchorElement>, authorString: string;
+  clientId: "793071505327652864"
+});
 
-  //* Author tags more than one =>
-  if (authors.length > 1) {
-    //* Convert to js array for .map function
-    authorsArray = Array.from(authors);
+interface langStrings {
+  play: string;
+  pause: string;
+  browsing: string;
+  listening: string;
+}
 
-    //* Build output string
-    authorString = `${authorsArray
-      .slice(0, authorsArray.length)
-      .map((a) => a.innerText)
-      .join(", ")}`;
-  } else
-    authorString = (document.querySelector(
+const getLanguages = async () => {
+    return presence.getStrings(
+      {
+        play: "presence.playback.playing",
+        pause: "presence.playback.paused",
```

Comment:
Please use the general strings now.
"general.playing"
"general.paused"

Reply:
oof didn't notice